presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastoreClient.java

```diff
@@ -121,6 +121,13 @@ public Database getDatabase(String dbName)
         return client.get_table_names_by_filter(databaseName, filter, (short) -1);
     }
 
+    @Override
+    public List<String> getTableNamesByType(String databaseName, String tablePattern, String tableType)
```

Comment:
We could expose this call without `tablePattern` param. There is no value in it.